lib/tokens/index.js

```diff
@@ -8,7 +8,7 @@ const error = require('../error')
 
 module.exports = (log, config) => {
   config = config || {}
-  const lifetimes = config.tokenLifetimes || {
+  const lifetimes = config.tokenLifetimes = config.tokenLifetimes || {
```

Comment:
Is it worth trying to import the defaults from the config module somehow, rather than duplicating?